.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+When filing an issue, please fill in the requested fields as thoroughly as you can and then delete this comment
+-->
+
+* Description of issue
+
+    description
+
+* `python -c "import sys; print(sys.version)"` output
+
+    ```
+    replace with output
+    ```
+
+* `python -m pip list` output
+
+    ```
+    replace with output
+    ```
+
+* Diver and version (this is the installed driver version, not the Python API version)
+
+    I.e. NI-DMM, 18.0
+
+* Operating system, version, and bitness
+
+    I.e. Windows 10 Pro 1803, x64
+
+* Steps to reproduce issue
+
+    This can be a simple python script that will reproduce the issue, or a list of steps, or anything else to
```

Comment:
Capitalize Python.
We should reword to encourage a code snippet. You can even show them how to properly use markdown.

```
   ''' python
   # Your code here
   '''
```

Reply:
Changed